Lib/test/test_os.py

```diff
@@ -179,6 +179,13 @@ def test_tmpnam(self):
             else:
                 self.check_tempfile(name)
 
+    @unittest.skipUnless(sys.platform == 'win32', 'tests windows behaviour')
+    def test_listdir_extended_length_path(self):
+        """See https://bugs.python.org/issue32539"""
+        dirname = '\\\\?\\' + os.path.abspath(test_support.TESTFN)
+        result = os.listdir(dirname)
```

Comment:
Does this directory exist? What if run `python -m test -m test_listdir_extended_length_path test_os`?

Reply:
it would crash otherwise, it's set up as part of this code: https://github.com/python/cpython/blob/ab95b3074ee43098edf3f23b07fb18ef57ee614d/Lib/test/test_os.py#L71-L73